src/libmongoc/doc/mongoc_server_description_hello_response.rst

```diff
@@ -0,0 +1,34 @@
+:man_page: mongoc_server_description_hello_response
+
+mongoc_server_description_hello_response()
```

Comment:
Alternatively, we can name this `mongoc_server_description_last_hello_response` to better indicate what this returns.

Reply:
Renamed to `mongoc_server_description_last_hello_response`. Changing this to be immutable sounds sensible.